firstvoices/backend/serializers/base_serializers.py

```diff
@@ -113,7 +115,22 @@ def validate(self, attrs):
         return super().validate(attrs)
 
 
-class LinkedSiteSerializer(serializers.HyperlinkedModelSerializer):
+class ReadOnlyVisibilityFieldMixin:
+    """
+    A mixin for ModelSerializers that provides a read-only visibility field.
+    """
+
+    visibility = serializers.SerializerMethodField(read_only=True)
+
+    @staticmethod
+    @extend_schema_field(OpenApiTypes.STR)
+    def get_visibility(instance):
+        return instance.get_visibility_display().lower()
+
+
+class LinkedSiteSerializer(
+    serializers.HyperlinkedModelSerializer, ReadOnlyVisibilityFieldMixin
```

Comment:
Put the mixin first in the list please. Base class should always be last

Reply:
Ah shoot sorry, will fix!

Reply:
Same for a few other serializers as well